popup.js

```diff
@@ -111,28 +143,28 @@ function nominationTool( e ) {
   data[DATABASE_ID] = databaseID;
   data[COMMMENT_ID] = commentID;
 
-  // Do GET call to post to Google Form and open new tab
-  $.get( {
-    url: GOOGLE_FORMS_URL,
-    data,
-    success: function( res ) {
-      $( '#success' ).html( "Success!" );
+  $.get(GOOGLE_FORMS_URL, data)
+    .then(() => {
+      $('#save').prop('disabled', false);
+
+      showStatus('success', 'URL submitted. Thanks!');
+      // @todo Why are we reloading the popup here? To reset some fields?
```

Comment:
sure.  I was just fooling aorund with it. 1000ms is too short I htink, mroe like 5000?